kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -657,4 +657,13 @@ object KyuubiConf {
     .version("1.1.1")
     .stringConf
     .createOptional
+
+  val ENGINE_SINGLE_SESSION =
+    buildConf("engine.single.spark.session")
```

Comment:
```
val ENGINE_SINGLE_SESSION =
    buildConf("engine.single.session")
```
or 
```
val ENGINE_SINGLE_SPARK_SESSION =
    buildConf("engine.single.spark.session")
```
? cc @pan3793

Reply:
hi @wForget  could you keep it consistent for the VARIABLE name and config entry name?

Reply:
Ok thanks, i will fix it